spec/abstract_adapter_spec.rb

```diff
@@ -20,10 +20,9 @@
       end
     end
 
-    it "does not create deprecation" do
-      expect_not_deprecated {
-        connection.tables_without_deprecation
-      }
+    it "does not cause deprecation" do
+      expect(ActiveSupport::Deprecation).not_to receive(:warn)
```

Comment:
I could have sworn I tried this and it didn't work for some reason, so I went with the cumbersome helper.  Can't imagine what the trouble was, and of course this way is much cleaner.

...though, have you verified that if you just called `connection.tables` the spec *would* fail, at least in some cases?  (That's the problem with `expect ... not_to` tests, it's hard to know if they're actually testing anything!)